SPEC.md

```diff
@@ -211,6 +211,65 @@ Plugins may define additional fields that they accept and may generate an error
 }
 ```
 
+### Network Configuration Lists
+
+Network configuration lists provide a mechanism to run multiple CNI plugins for a single container in a defined order, passing the result of each plugin to the next plugin.
```

Comment:
Perhaps, but if this feature isn't useful for Kubernetes, I'd argue the feature isn't right.